### PR TITLE
End point to register new applicants & trigger mail to applicant

### PR DESCRIPTION
**Pull request by PrateekxCodes**

### CODE REVIEW

**Comment by abhinavsingh06** [comment]
Tested and looks good.

File: routes/user.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// on first login reset Password
+router.post("/:hashMail", async (req, res) => {
+  let { password } = req.body;
+  let { hashMail } = req.params;
+  if (hashMail) {
+    try {
+      const user = await User.findOne({ hashMail });
+      user.password = password;
+      delete user.hashMail;
+      const UpdatedUser = await user.save();
```
Comment (prank7) [inline routes/user.js]: Should be updatedUser and not UpdatedUser. Caps is used only for global variables.

File: routes/user.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      return res.status(201).json({ status: true, UpdatedUser });
```
Comment (prank7) [inline routes/user.js]: The response should be 

`res.status(201).json({ status: true, user: updatedUser });`

Always send consistent key in response object.

**Review by prank7 — CHANGES_REQUESTED** [review]
@prarora98 @Ckbhatia please check the review, correct and update the PR

**Comment by prank7** [comment]
close #11

**Pull request merged**